core/pkg/service/flag-evaluation/flag_evaluator_test.go

```diff
@@ -903,3 +903,50 @@ func getMetricReader() (*telemetry.MetricsRecorder, metric.Reader) {
 	rs := resource.NewWithAttributes("testSchema")
 	return telemetry.NewOTelRecorder(exp, rs, "testSvc"), exp
 }
+
+// TestFlag_Evaluation_ErrorCodes test validate error mapping from known errors to connect.
+// Code and avoid accidental changes. This is essential as SDK implementations rely on connect.
+// Code to differentiate GRPC errors vs Flag errors. For any change, we must change respective SDK changes
```

Comment:
```suggestion
// TestFlag_Evaluation_ErrorCodes test validate error mapping from known errors to connect.
// Code and avoid accidental changes. This is essential as provider implementations rely on connect.
// Code to differentiate gRPC errors vs Flag errors. Changes must be absorbed by flagd providers.
```

Reply:
The capitalization here is a bit odd. Maybe a block comment would be better?

Reply:
Fixed with the latest commit. Thank you pointing this out